### PR TITLE
issues-47: fixed non-object exception issue

### LINKED ISSUE #47: this package is not working on Laravel 5.2.x
when I add "stormpath/laravel": "^0.4" to composer.json file, run ''composer update" from my command line, I have received Runtime Exception.

I think so, this package not compatible with new Laravel 5.2.x

screen shot 2016-05-02 at 16 42 48

### PR DESCRIPTION
**Pull request by nurettintopal**
You can get more information from issue https://github.com/stormpath/stormpath-laravel/issues/47

Fixes #47

### CODE REVIEW

File: src/Support/StormpathLaravelServiceProvider.php [hunk 1]
```diff
@@ -109,8 +109,8 @@ private function warmResources()
                 'href' => $mapping->accountStore->href,
                 'name' => $mapping->accountStore->name,
                 'provider' => [
-                    'href' => $mapping->accountStore->provider->href,
-                    'providerId' => $mapping->accountStore->provider->providerId,
+                  'href' => isset($mapping->accountStore->provider->href) ? $mapping->accountStore->provider->href : null,
+                  'providerId' => isset($mapping->accountStore->provider->providerId) ? $mapping->accountStore->provider->providerId : null,
```
Comment (bretterer) [inline src/Support/StormpathLaravelServiceProvider.php]: I'm a little concerned with this.  AFAIK, there should never be a case where these two values would not be set.

Comment (bretterer) [inline src/Support/StormpathLaravelServiceProvider.php]: I am actually wrong... Provider only lives on directories which means that provider would not exist.  What should be done here is set the acountStoreArray and check to see if the `$mapping->accountStore->provider`.  if it does, then we attach the provider array to it.  if not, provider array should not be included

Comment (nurettintopal) [inline src/Support/StormpathLaravelServiceProvider.php]: it is OK. I updated my pull request. please check it.

**Comment by nurettintopal** [comment]
@bretterer 

what do you think about this pull request? is it OK?

**Comment by bretterer** [comment]
Hey @nurettintopal .  Sorry for the delay.  I hope to have this merged in and a new release start of next week for the Laravel plugin.  Your PR looks ok, I may tweak a few items for a little more safety checking.

**Pull request merged**